internal/replacements/replace_failed_process_groups.go

```diff
@@ -73,6 +75,12 @@ func ReplaceFailedProcessGroups(log logr.Logger, cluster *fdbv1beta2.FoundationD
 			continue
 		}
 
+		for _, targets := range cluster.GetCrashLoopContainerProcessGroups() {
```

Comment:
Should we move the `cluster.GetCrashLoopContainerProcessGroups()` statement out of the for loop to calculate the map only once instead of multiple times? In the current implementation we calculate the same map for every iteration.